src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/EventAuditRepository.kt

```diff
@@ -22,12 +23,11 @@ interface EventAuditRepository : JpaRepository<EventAudit, Long> {
   fun findLastBookedVisitEventByBookingReference(bookingReference: String): EventAudit?
 
   @Query(
-    "SELECT actioned_by FROM event_audit " +
-      "WHERE booking_reference = :bookingReference AND type in ('UPDATED_VISIT','BOOKED_VISIT') " +
-      "ORDER BY id DESC LIMIT 1 ",
-    nativeQuery = true,
+    "SELECT ea.actionedBy FROM EventAudit ea " +
```

Comment:
Question: why did you change this from native SQL to JPQL?

Reply:
it was an easy way to return the actionedBy with out having to do a join